portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationCreateSegment.testcase

```diff
@@ -819,6 +819,57 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
+	@description = "This is a use case for LPS-131815"
+	@priority = "5"
+	test SegmentationCheckSpecialCharacters {
+		property portal.acceptance = "true";
+
+		JSONUser.addUser(
+			userEmailAddress = "u1@liferay.com",
+			userFirstName = "User",
+			jobTitle = "Dev",
+			userLastName = "1",
+			userScreenName = "u1"
+			);
+
+		JSONUser.addUser(
+			userEmailAddress = "u2@liferay.com",
+			userFirstName = "User",
+			jobTitle = "%Dev",
+			userLastName = "2",
+			userScreenName = "u2"
+			);
+
+		LexiconEntry.gotoAdd();
```

Comment:
Just for information, I was not able to create user using JSONSegmentsentry.addSegment, because it's not accepting special characters, needed for this test.